examples/sparse/hypergraphatt.py

```diff
@@ -2,13 +2,17 @@
 Hypergraph Convolution and Hypergraph Attention
 (https://arxiv.org/pdf/1901.08150.pdf).
 """
-import dgl
-import dgl.mock_sparse as dglsp
+import argparse
+
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
-from torchmetrics.functional import accuracy
 import tqdm
+from torchmetrics.functional import accuracy
+
+import dgl.mock_sparse as dglsp
```

Comment:
We should be able to migrate this example to `mock_sparse2`.

Reply:
Yes, I've done that locally. Waiting for the bspmm/bsddmm PR to be merged and will push all the changes later.